stubs/flake8-builtins/flake8_builtins.pyi

```diff
@@ -1,12 +1,38 @@
 import ast
-from _typeshed import Incomplete
-from collections.abc import Generator
-from typing import Any, ClassVar
+from collections.abc import Iterator
+from typing import ClassVar
+from typing_extensions import TypeAlias
+
+_Error: TypeAlias = tuple[int, int, str, type[BuiltinsChecker]]
 
 class BuiltinsChecker:
     name: ClassVar[str]
     version: ClassVar[str]
-    def __init__(self, tree: ast.AST, filename: str) -> None: ...
-    def run(self) -> Generator[tuple[int, int, str, type[Any]], None, None]: ...
+    assign_msg: ClassVar[str]
+    argument_msg: ClassVar[str]
+    class_attribute_msg: ClassVar[str]
+    import_msg: ClassVar[str]
+    module_name_msg: ClassVar[str]
 
-def __getattr__(name: str) -> Incomplete: ...  # incomplete (other attributes are normally not accessed)
+    names: ClassVar[list[str]]
+    ignore_list: ClassVar[set[str]]
+    ignored_module_names: ClassVar[set[str]]
+
+    def __init__(self, tree: ast.AST, filename: str) -> None: ...
+    @classmethod
+    def add_options(cls, option_manager) -> None: ...
+    @classmethod
+    def parse_options(cls, options) -> None: ...
```

Comment:
Since these are the only two missing params:
```suggestion
    def add_options(cls, option_manager: OptionManager) -> None: ...
    @classmethod
    def parse_options(cls, options: Namespace) -> None: ...
```

Where
```py
from argparse import Namespace
from flake8.options.manager import OptionManager
```

Ref: https://flake8.pycqa.org/en/latest/plugin-development/plugin-parameters.html#registering-options

Reply:
You beat me to it :)